routers/routes/routes.go

```diff
@@ -722,6 +722,8 @@ func RegisterRoutes(m *macaron.Macaron) {
 			m.Post("/labels", reqRepoIssuesOrPullsWriter, repo.UpdateIssueLabel)
 			m.Post("/milestone", reqRepoIssuesOrPullsWriter, repo.UpdateIssueMilestone)
 			m.Post("/assignee", reqRepoIssuesOrPullsWriter, repo.UpdateIssueAssignee)
+			m.Post("/request_review", reqRepoIssuesOrPullsWriter, repo.UpdatePullReviewRequest)
```

Comment:
It seems that this function doesn't allow the PR creator to request reviews unless they have special permissions.

Reply:
my thought: same as github style , a PR creator who don't have write permissions can only ask for review again after an review is posted. And the Review Request is just a official notify to reviewers, It's not a special permissions like Assignees.

Reply:
OK, I didn't know Github worked like that. It would be cool to have the option (in `app.ini`) to allow normal users to ask for reviews, because Gitea is used a lot in enterprise environments and there the requirements are a little different than in open source projects. This however could be implemented later in another PR so this one doesn't get too complicated. 👍